### ISSUE TITLE
Supporting remote file in configuration 

### ISSUE THREAD

**Issue by yonbav** [Ozsie/detekt-maven-plugin#132]
Hi everyone.
First of all, thanks for the great plugin  :)
It seems that the configuration is only supporting local files.
which means this will work
```
            <configuration>
              <config>./detekt-with-complexity-report.yml</config>
              <failBuildOnMaxIssuesReached>false</failBuildOnMaxIssuesReached>
            </configuration>
```
But this will not work
```
            <configuration>
              <config>https://myArtifactory.com/artifactory/detekt/detekt-with-complexity-report.yml</config>
              <failBuildOnMaxIssuesReached>false</failBuildOnMaxIssuesReached>
            </configuration>
```
can you please add support in remote configuration files as well?

**Comment by Ozsie** [comment]
Hi, I'm not sure if I want to add this. As far as I can tell, detekt it self does not support this and this plugin just passes along the properties to the detekt CLI.

**Comment by yonbav** [comment]
That Makes sense, although it shouldn't be too hard to copy the file from a remote location locally and use it.  If that's fine with you I can create a PR that does that.
If not that is 100% fine, just let me know and I will close the issue.

**Comment by Ozsie** [comment]
Of course, create a PR and I'll have a look at it.

**Comment by yonbav** [comment]
Hi @Ozsie, I created the code. In order to create the PR I need a remote location for the unit tests.
is this project has a remote location somewhere? It can even be google drive or something like that?

**Comment by Ozsie** [comment]
> Hi @Ozsie, I created the code. In order to create the PR I need a remote location for the unit tests.
> is this project has a remote location somewhere? It can even be a file in this github repo?

No, there is no such remote location, however, couldn't you use GitHub for this? For example you could include the config file in a `test/resources` directory and use something like this in the test `https://raw.githubusercontent.com/Ozsie/detekt-maven-plugin/master/src/test/resources/remote-config.yml`

If you put it in your PR branch, and verify that it works, I can update the test case when merging the PR.

**Comment by yonbav** [comment]
Yes, Great I will do that :)
I just wanted to make sure you don't have some specific standard for that.
it has to be in directory X

**Comment by yonbav** [comment]
@Ozsie I opened this [PR](https://github.com/Ozsie/detekt-maven-plugin/pull/139), regarding that subject.

**Comment by Ozsie** [comment]
@yonbav I have just released version 1.19.0.1 with your PR merged. It might take a short while for the release to propagate through all repositories, but it should be available form maven central.
